web_widget_image_webcam/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=========================
+Web Widget - WebCam Image
+=========================
+
+This module was written to extend the functionality of the image widget and allows to take snap with WebCam.
+
+Configuration
+=============
+
+By default, the module is worked with every major browsers.
+
+An important note for Chrome 47+ users, this module only works with websites those have SSL / HTTPS.
```

Comment:
An important note for Chrome 47+ users - this module only works with websites delivered over SSL / HTTPS.